apps/web/src/actions/rewards/fetchClaimedRewardsAction.ts

```diff
@@ -0,0 +1,13 @@
+'use server'
+
+import { ClaimedRewardsRepository } from '@latitude-data/core/repositories'
+
+import { authProcedure } from '../procedures'
+
+export const fetchClaimedRewardsAction = authProcedure
```

Comment:
We should move away from action for `GET` requests and use API calls. We get HTTP caching.